charts/all-in-one/templates/ninechronicles-util-backend.yaml

```diff
@@ -0,0 +1,59 @@
+{{ if .Values.ninechroniclesUtilBackend.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ninechronicles-util-backend
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+  name: ninechronicles-util-backend
+  namespace: {{ $.Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ninechronicles-util-backend
+  template:
+    metadata:
+      labels:
+        app: ninechronicles-util-backend
+    spec:
+      containers:
+        - image: {{ $.Values.ninechroniclesUtilBackend.image.repository }}:{{ $.Values.ninechroniclesUtilBackend.image.tag }}
+          name: ninechronicles-util-backend
+          env:
+            - name: StateService__HeadlessEndpoint
+              value: {{ $.Values.ninechroniclesUtilBackend.headlessEndpoint }}
+            - name: ASPNETCORE_URLS
+              value: http://*:3000
+      {{- with $.Values.ninechroniclesUtilBackend.nodeSelector }}
```

Comment:
`$.Values.ninechroniclesUtilBackend.nodeSelector` value in `9c-main/multiplanetary/network/heimdall.yaml` is missing 🙏

Reply:
Since it uses `with` keyword, the `.nodeSelector` field is optional. Is it required for operation? If you worried the architecture (amd64 or arm64), the 9cutil-backend support both them.

Reply:
Ah, ok that's fine. I was also worried that it will use any type of node available which would be problematic if `ninechronicles-util-backend` needs more resources than for example, the `m5.large` instance type.